docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,219 @@
+
+.. code-block:: python3
+
+    if administrator_role > manager_role:
+        # The administrator role is greater in the hierarchy than manager.
+
+One small caveat to note with the discord permissions system, is that in the case of overwrites, explicit allow will always overwrite explicit deny.
+
+What I mean by this, is the following:
+
+Take a scenario where you have a text channel, that gives the ``@everyone`` role the ``send_messages`` overwrite set to ``True`` (or a green check-mark in the UI).
+If you then have a Muted role, that is supposed to disallow members from sending messages, but that role has the ``send_messages`` overwrite set to ``False`` (or a red cross in the UI)
+then members with that Muted role **can** still send  messages within this channel.
+
+The solution to this issue is to set the ``@everyone`` role overwrites for ``send_messages`` to ``None`` (or a grey strike in the UI).
+This allows for permissions to be cascaded to it.
+
+.. image:: /images/guide/topics/permissions/green_check.png
+    :scale: 80%
+
+.. image:: /images/guide/topics/permissions/red_cross.png
+    :scale: 80%
+
+.. image:: /images/guide/topics/permissions/grey_strike.png
+    :scale: 80%
+
+
+Using permissions within discord.py
+------------------------------------
+
+discord.py has two primary ways of interaction with the permissions of a guild or channels:
+
+- :class:`~Permissions` for interacting with general permissions for roles and channels.
+- :class:`~PermissionOverwrite` for interacting with the overwrites on a guild channel.
+
+Let's use both of these now in actual examples.
+
+Permission values
+------------------
+
+A small preface to this, is that discord themselves represent the permissions system as a bit field.
+They have a great in-depth `explanation to this <https://discord.com/developers/docs/topics/permissions>`_ on their API documentation page.
+
+If you desire to use permission integers within discord.py, you can do it similar to the below:
+
+.. code-block:: python3
+
+    # If you wanted to create a new admin role, that has the `Administrator` permission, you can do it like so:
+    permissions = discord.Permissions(administrator=True)
+
+    # However, the underlying value for this permission (or `bit` as the API docs explain it) can be viewed by:
+    permissions.value
+    >>> 8
+
+    # You can also instantiate a Permissions class with an integer that represents the bit(s) you want to provide, like so:
+    permissions = discord.Permissions(268435634)
+    # This Permissions instance has the `view_audit_log`, `manage_server`, `manage_roles`, `manage_channels` and `kick_members` permissions.
+
+
+Using Permissions to create a new role
+---------------------------------------
+
+So we want to make a role for a new game in our guild, that we can add to users to ping them for news and such. Let's do that now:
```

Comment:
Our examples should try to avoid being vague wherever possible.
```suggestion
Let's imagine we have a server where we talk about the latest games.

We want to make it so that when a new game comes out, we create a role that people can get to receive the latest news about that game.

Let's start out with making the role:
```